Service graph & span metrics connectors

ralongit opened the pull request:
- Upgrade `logzio-monitoring` to v3.3.0
  - Upgrade `logzio-k8s-telemetry` to v2.2.0
    - Upgraded SPM collector image to version `0.80.0`.
    - Added service graph connector metrics.
      - `serviceGraph.enabled` option.
    - Refactored span metrics processor to a connector.
    - Added metrics transform processor to modify the data for Logz.io backwards compatibility.

ralongit committed:
Service graph & span metrics connectors

- Upgraded SPM image to version 0.80.0
- Upgrade chart version to 1.3.0
- Added Service graph connector metrics
- Refactored Span metrics processor to a connector
  - Added metrics transform processor to modify the data to support Logz.io backwards compatibility.

tamir-michaeli reviewed (approved):
great work!
Please update the changelog :)

ralongit committed:
Version 1.3.0 readme update

ralongit committed:
Add system related dimensions to SPM

- Added db.system & messaging.system dimensions to SPM.

ralongit committed:
Merge pull request #344 from logzio/add-system-labels-to-spm

Add system related dimensions to SPM

ralongit committed:
Merge branch 'master' of https://github.com/logzio/logzio-helm into add-servicegraphconnector

ralongit committed:
Add service graph metrics flag

- Upgrade `logzio-monitoring` to v3.3.0
- Upgrade `logzio-k8s-telemetry` to v2.2.0
  - Upgraded SPM collector image to version `0.80.0`.
  - Added service graph connector metrics.
    - `serviceGraph.enabled` option.
  - Refactored span metrics processor to a connector.
    - Added metrics transform processor to modify the data for Logz.io backwards compatibility.

tamir-michaeli committed:
Update charts/logzio-telemetry/values.yaml

Removed whitespaces

tamir-michaeli commented on charts/logzio-telemetry/Chart.yaml:
```diff
@@ -30,7 +30,8 @@ dependencies:
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
+
```
```suggestion
```

tamir-michaeli committed:
Update charts/logzio-telemetry/Chart.yaml

removed blank line

tamir-michaeli reviewed (approved):
Great job!

mirii1994 commented on charts/logzio-telemetry/Chart.yaml:
```diff
@@ -30,7 +30,7 @@ dependencies:
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
+version: 2.2.0
```
I think we should bump to 3.x, since we're switching from the processor to the connector, which changes the metrics name if i'm not mistaken.
We also need to add a message about this - that when upgrading it will cause metrics names changes.
